src/points/PointsHistoryBottomSheet.tsx

```diff
@@ -64,6 +67,18 @@ function PointsHistoryBottomSheet({ forwardedRef }: Props) {
   const insets = useSafeAreaInsets()
 
   const onFetchMoreHistory = () => {
+    if (
+      !hasNextPage ||
+      pointsHistoryStatus === 'loadingNextPage' ||
+      pointsHistoryStatus === 'loadingFirstPage'
+    ) {
+      // prevent fetching more history if there is no next page. onEndReached
+      // will continue to fire if this is not checked. onEndReached also does
+      // not have a throttle so prevent fetching more history when there is
+      // already a request in flight.
+      return
```

Comment:
specifically on my simulator, there were not enough total history items to stop the onEndReached from firing continuously in the background